projects/igniteui-angular/src/lib/grids/grid/grid.component.spec.ts

```diff
@@ -2202,6 +2203,61 @@ describe('IgxGrid Component Tests', () => {
                 expect(editedCell.column.field).toEqual('Downloads');
                 expect(editedCell.inEditMode).toEqual(true);
             }));
+
+            it(`Should update row changes when focus overlay buttons on tabbing`, (async() => {
+                let currentEditCell: IgxGridCellComponent;
+                const targetCell = fixture.componentInstance.focusGridCell(0, 'Downloads');
+                fixture.detectChanges();
+                grid.parentVirtDir.getHorizontalScroll().scrollLeft = 0;
+                await wait(300);
+
+                // go to first editable cell
+                targetCell.onKeydownEnterEditMode();
+                fixture.detectChanges();
+                await wait(300);
+
+                // change first editable cell value
+                targetCell.editValue = '500';
+                fixture.detectChanges();
+                await wait(300);
+
+                // go to Done
+                fixture.componentInstance.moveNext(true);
+                fixture.detectChanges();
+                await wait(300);
+
+                let overlayText = document.getElementsByClassName(BANNER_TEXT)[0] as HTMLElement;
+                expect(overlayText.textContent.trim()).toBe('You have 1 changes in this row');
+
+                // focus Cancel
+                const bannerRowElement = fixture.debugElement.query(By.css('.igx-banner__row')).nativeElement;
+                const cancelButtonElement = bannerRowElement.firstElementChild;
+                cancelButtonElement.focus();
+                fixture.detectChanges();
+                await wait(300);
+
+                // go to last editable cell
+                document.activeElement.dispatchEvent(new KeyboardEvent('keydown', { key: 'tab', code: 'tab', shiftKey: true }));
```

Comment:
You have to use:
 UIInteractions.triggerKeyDownWithBlur('tab',  document.activeElement, true, false, true);

Reply:
fixed :)